.github/autobuild/linux_deb.sh

```diff
@@ -50,10 +53,19 @@ setup_cross_compiler() {
     sudo update-alternatives --install "/usr/bin/${ABI_NAME}-g++" g++ "/usr/bin/${ABI_NAME}-g++-${GCC_VERSION}" 10
     sudo update-alternatives --install "/usr/bin/${ABI_NAME}-gcc" gcc "/usr/bin/${ABI_NAME}-gcc-${GCC_VERSION}" 10
 
-    if [[ "${TARGET_ARCH}" == armhf ]]; then
-        # Ubuntu's Qt version only ships a profile for gnueabi, but not for gnueabihf. Therefore, build a custom one:
+    # Ubuntu's Qt version only ships a profile for gnueabi, but not for gnueabihf or aarch64. Therefore, build a custom one:
+    local replace=""
+    case "${TARGET_ARCH}" in
```

Comment:
Actually...  Lines 10 to 22 are dealing with this already.  It's probably clearer to set things up back there rather than have two separate `case` statements.

Reply:
Done.